### PR TITLE
[PREVIEW] DIV-2904 

### PR DESCRIPTION
**Pull request by niyioyelade**
Content change in mini petition for financial orders - Template change and integration test templates updated

This contains html template changes for the generation of Mini Petition.
Integration test input and pdf output files have also been updated accordingly.

This work also contains the tech debt of upgrading div-doc-gen to SpringBoot 2.

# Description

[DIV-2904 - Content Change](https://tools.hmcts.net/jira/browse/DIV-2904)

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -147,30 +190,20 @@ dependencies {
 
     compileOnly group: 'org.projectlombok', name: 'lombok', version: versions.lombok
     testCompile group: 'com.github.tomakehurst', name:'wiremock', version: versions.wiremockVersion
-    testCompile group: 'org.powermock', name:'powermock-api-mockito', version:versions.powerMock
+//    testCompile group: 'org.springframework.cloud', name: 'spring-cloud-contract-wiremock', version: '2.0.0.M5'
```
Comment (mthava) [inline build.gradle]: if this is not needed please remove it

File: src/integrationTest/java/uk/gov/hmcts/reform/divorce/ResourceLoader.java [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 public class ResourceLoader {
 
-    public static String loadJSON(final String filePath) throws Exception {
+    public static String loadJson(final String filePath) throws Exception {
```
Comment (mthava) [inline src/integrationTest/java/uk/gov/hmcts/reform/divorce/ResourceLoader.java]: Nice 👍

File: src/main/resources/application.yml [hunk 3]
```diff
@@ -3,9 +3,9 @@ server:
 
 service:
   pdf-service:
-    uri: ${PDF_SERVICE_BASEURL:http://localhost:5500}/pdfs
+    uri: ${PDF_SERVICE_BASEURL:http://localhost:4502}/pdfs
     health:
-      uri: ${PDF_SERVICE_BASEURL:http://localhost:5500}/health
+      uri: ${PDF_SERVICE_BASEURL:http://localhost:4502}/health
```
Comment (mthava) [inline src/main/resources/application.yml]: Why are you changing the port number? This could be the reason for the healthcheck failure

Comment (niyioyelade) [reply src/main/resources/application.yml]: It is not the reason for health check failure, rather it was the temporary fix I needed to be able to see the actually issue with the health check. I have now returned it to the old port.

File: src/integrationTest/java/uk/gov/hmcts/reform/divorce/ResourceLoader.java (hunk 2 above)
Comment (niyioyelade) [reply src/integrationTest/java/uk/gov/hmcts/reform/divorce/ResourceLoader.java]: Thanks

File: src/main/resources/data/templates/divorceminipetition.html [hunk 4]
```diff
@@ -4,32 +4,32 @@
         /* Open Sans Light */
         @font-face {
             font-family: 'Open Sans';
-            format('woff');
+        format('woff');
```
Comment (tchow8) [inline src/main/resources/data/templates/divorceminipetition.html]: Nothing major but on github for me this line looks like it has the wrong indentation.

Comment (niyioyelade) [reply src/main/resources/data/templates/divorceminipetition.html]: That's right. I have fixed it.

**Comment by MOsinloye** [comment]
This has been tested and passed.
Signed off by Oonagh.
All three scenarios of FO wordings are covered and pdfs attached to DIV- 2904.
Ready for merge

**Comment by nsingamsetty** [comment]
Merging this PR onbehalf of Michael

**Pull request merged**